tests/main/interfaces-password-manager-service/task.yaml

```diff
@@ -1,36 +1,28 @@
 summary: Ensure that the password-manager-service interface works
 
-# Only test on classic systems with AppArmor DBus mediation
-systems: [ ubuntu-1* ]
+systems:
+    - ubuntu-16.04-*
+    - ubuntu-18.04-*
+    - ubuntu-20.04-*
```

Comment:
How about: `- ubuntu-2*` here ? To ensure it runs on all future ubuntu 20,22,24 etc?